tests/cli-v1.rs

```diff
@@ -109,9 +109,55 @@ fn list_toolchains() {
             &["rustup", "toolchain", "list", "--verbose"],
             "(default)\t",
         );
+        #[cfg(windows)]
+            expect_stdout_ok(
+            config,
+            &["rustup", "toolchain", "list", "-v"],
+            "\\toolchains\\nightly-x86",
+        );
+        #[cfg(windows)]
+            expect_stdout_ok(
+            config,
+            &["rustup", "toolchain", "list", "--verbose"],
+            "\\toolchains\\nightly-x86",
+        );
```

Comment:
The `-v`/`--verbose` thing doesn't really need checking here, pick one and test that, no need for both.